### DISCUSSION TITLE
Arctic Code Vault Contributor

### DISCUSSION THREAD

**Discussion by Nyykol** [Schweinepriester/github-profile-achievements#1] [Q&A] [answered]
Can I still get the Arctic Code Vault Contributor achievement at 2022 or rn?

**Comment by Schweinepriester** [comment, marked as answer]
Unfortunately no, as the deadline was in the past.

https://archiveprogram.github.com/arctic-vault/ defines it:

> The 02/02/2020 snapshot archived in the GitHub Arctic Code Vault swept up every active public GitHub repository. It included every repo with any commits between the announcement at GitHub Universe on November 13th and 02/02/2020; every repo with at least 1 star and any commits from the year before the snapshot; and all repos with at least 250 stars. (It also included gh-pages for those repositories.) The snapshot consists of the HEAD of the default branch of each repository, minus any binaries larger than 100KB in size. (Repos with 250+ stars retained their binaries.)

**Reply by Nyykol** [reply]
Ok,ty

**Reply by Nyykol** [reply]
So the only achievement I can get is only "GitHub Sponsor"?

**Reply by Schweinepriester** [reply]
Yes, as far as I know (and official documentation says).

**Reply by Nyykol** [reply]
ok

**Reply by farkon00** [reply]
> So the only achievement I can get is only "GitHub Sponsor"?

Not anymore. Now i think you already may have one. And if you dont, you can earn one of new ones(added 9.06.2022). Simplest way to get one of them is making and merging pull request.

**Reply by arbso** [reply]
So answering this will give a badge?

**Reply by ydainna** [reply]
> Donc, répondre à cela donnera un badge ?

yes

**Comment by muhammadqazi** [comment]
How can I get Galaxy Brain?

**Reply by farkon00** [reply]
> How can I get Galaxy Brain?

I think you can create few disscusions yourself and answer them. But this is cheating and there is no fun in this solution

**Reply by Schweinepriester** [reply]
@farkon00 I actually think it doesn't work if you select yourself as the answer.

**Reply by farkon00** [reply]
> @farkon00 I actually think it doesn't work if you select yourself as the answer.

Still prs to my own repos count for pull shark. So why galaxy brain wont work with your own disscusion

**Reply by muhammadqazi** [reply]
looks like someone has to check that 😅

**Reply by farkon00** [reply]
> looks like someone has to check that 😅

Check it then. I dont want to give myself badge for free

**Reply by Schweinepriester** [reply]
@farkon00 Check my profile:

> 2 discussions answered

It's this one and the other one is https://github.com/Schweinepriester/beets-itunesimport/discussions/3 (which I selected myself way back when).
Yet, no Galaxy Brain…

**Reply by ydainna** [reply]
it is necessary to answer the discussions

**Reply by khushnoodasif** [reply]
Does this count as my discussion.

**Reply by mMarzeta** [reply]
I'm going to check it. This is basically my second answer to the discussion. If a badge will appear on my profile then it works.

**Reply by mMarzeta** [reply]
Okey, I haven't received the badge. Probably it's updated periodically with some reasonable frequency like once in a week or so.

**Reply by farkon00** [reply]
> I'm going to check it. This is basically my second answer to the discussion. If a badge will appear on my profile then it works.

I think your comment must be picked as answer by author.

**Reply by melvincwng** [reply]
@muhammadqazi , to get the Galaxy Brain achievement badge:
- Participate in Github Discussions (like this one) and get 2 of your answers being 'Marked as answer' by the person asking the question.
- You can unlock different tiers of the Galaxy Brain badge (bronze, silver, gold) as well - depending on how many answered discussions you have.

Reference: https://github.com/Schweinepriester/github-profile-achievements

**Reply by lgrachov** [reply]
You must reply on comment by other!

**Comment by jazzybruno** [comment]
The poeple who got that  badge had contribute to a repository in 2020 but now its over so you cat get it

**Reply by YouthguyZ** [reply]
really??

**Reply by melvincwng** [reply]
@YouthguyZ, yes that badge could only be obtained in 2020.

Hope this clarifies.
